spec/views/email_addresses/edit.html.erb_spec.rb

```diff
@@ -1,11 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe "email_addresses/edit", type: :view do
+  let(:person) { Person.create(first_name: "Bob", last_name: "Smith") }
+  let(:email_address) { EmailAddress.create(contact_id: person.id, contact_type: 'Person', address: "MyString") }
```

Comment:
Line is too long. [113/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.